src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/UkCountry.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation.utils;
+
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators.setErrorMsgToLocation;
+
+public enum UkCountry {
+    ENGLAND("England"),
+    NORTHERN_IRELAND("Northern Ireland"),
+    SCOTLAND("Scotland"),
+    WALES("Wales");
+
+    public final String countryName;
+
+    UkCountry(String country) {
+        this.countryName = country;
+    }
+
+    public static boolean isValid(String countryIn, String location, Errors errs, String loggingContext) {
```

Comment:
'countryIn' confuses me a bit. Does the 'In' represent an input parameter? Could just call this 'countryName'.

Reply:
It's what I have added for Countries, but we can come back to it.